### PR TITLE
add scoring pipeline test

### PR DESCRIPTION
**Pull request by grehx**

### CODE REVIEW

File: regression-tests/automation/run_tests.sh [hunk 1]
```diff
@@ -35,4 +35,4 @@ export SPARKTK_HOME=$sparktkpackage/
 echo "spark tk home"
 echo $SPARKTK_HOME
 
-py.test --boxed -n48 -v $MAINDIR/regression-tests
+py.test --boxed -n48 -v $MAINDIR/regression-tests -k naivebayes
```
Comment (lewisc) [inline regression-tests/automation/run_tests.sh]: revert this change

File: regression-tests/sparktkregtests/lib/scoring_utils.py [hunk 2]
```diff
@@ -22,26 +22,31 @@
...
-        # set port
+        self.pipeline = pipeline
+        self.pipeline_filename = pipeline_filename
         port_config = SafeConfigParser()
         filepath = os.path.abspath(os.path.join(
             config.root, "regression-tests", "sparktkregtests", "lib", "port.ini"))
         port_config.read(filepath)
         self.port = port_config.get('port', port_id)
-        self.scoring_process = None
+        #self.scoring_process = None
```
Comment (lewisc) [inline regression-tests/sparktkregtests/lib/scoring_utils.py]: is this supposed to be commented out? If it's not wanted delete it.

**Review by lewisc — CHANGES_REQUESTED** [review]

**Comment by lewisc** [comment]
Looks good

File: regression-tests/automation/requirements.txt [hunk 3]
```diff
@@ -16,3 +16,6 @@ pip
 pyopenssl
 ndg-httpsclient
 pyasn1
+flask
+flask_api
+kafka
```
Comment (blbarker) [inline regression-tests/automation/requirements.txt:21]: These 3 should not be added --I don't see them being used?

**Review by blbarker — DISMISSED** [review]

**Pull request merged**